src/diffusers/schedulers/scheduling_unclip.py

```diff
@@ -205,6 +261,8 @@ def step(
         prev_timestep: Optional[int] = None,
         generator=None,
         return_dict: bool = True,
+        # YiYi's TO-DO: batch_size argument for testing, remove this later
+        batch_size: Optional[int] = None,
```

Comment:
yes let's not forget to remove this

Reply:
@patrickvonplaten I think it might be easier to keep this argument because our  `randn_tensor()` function https://github.com/huggingface/diffusers/blob/main/src/diffusers/utils/torch_utils.py#L67 expects the `latent` tensor to be the same size as `batch_size`, and also the number of generators (if we pass a list of generators) 

if we are going to generate `latents` with `2 * batch_size` shape we will have to refactor the logic  - think it's unnecessary? 

let me know - if it's ok with you I will just remove this TO-DO comment